aldryn_newsblog/boilerplates/bootstrap3/templates/aldryn_newsblog/plugins/article_search.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n sekizai_tags %}
+
+{% addtoblock "js" %}
+<script>
+(function($){
+    $('.aldryn-newsblog-article-search').find('form').on('submit', function(evt){
+        var $form = $(this);
```

Comment:
I think we'll have to agree to disagree on this. a variable that starts with '$', to me, means: here is a jQuery object. This is a widely accepted convention.